### PR TITLE
Ensure we have icons for all channel types in use

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: src/vectoricon/index.ts [hunk 1]
```diff
@@ -34,38 +34,6 @@ export enum Icon {
...
-  channel_d3 = 'channel-whatsapp',
-  channel_ds = 'channel-discord',
-  channel_ex = 'zap',
-  channel_fb = 'channel-facebook',
-  channel_fba = 'channel-facebook',
-  channel_fc = 'channel-freshchat',
-  channel_fcm = 'channel-firebase',
-  channel_ig = 'channel-instagram',
-  channel_jc = 'channel-jiochat',
-  channel_jn = 'channel-junebug',
```
Comment (rowanseymour) [inline src/vectoricon/index.ts:48]: no longer a type

File: src/vectoricon/index.ts [hunk 2]
```diff
@@ -34,38 +34,6 @@ export enum Icon {
...
-  channel_kn = 'channel-kannel',
-  channel_kwa = 'channel-whatsapp',
-  channel_ln = 'channel-line',
-  channel_mg = 'channel-mtarget',
```
Comment (rowanseymour) [inline src/vectoricon/index.ts:52]: should be `_mt`

**Review by norkans7 — APPROVED** [review]

**Comment by norkans7** [comment]
We probably need to have a way to have default icon when this is not updated for a new channel type we add

**Comment by rowanseymour** [comment]
@norkans7 @ericnewcomer not sure how that could work.. was wondering about a RP unit test that looks inside the svg file for all the codes ?

**Comment by norkans7** [comment]
even that list is missing ZVS which is probably not a useful channel type now but could have a broken is the icon is missing

**Comment by rowanseymour** [comment]
@norkans7 I only included channel types that are in use on our deployments.. but you're right.. really we need all types that can appear on the claim page

**Comment by norkans7** [comment]
I guess the `temba-icon` should be able to display some default icon if the one we want is missing or take an alternative that can be always define depending on the context for these dynamic cases

**Comment by rowanseymour** [comment]
oops that already exists https://github.com/nyaruka/temba-components/blob/1d03e2af70584e5230f993ff9346d4cd75a7a0a9/src/vectoricon/VectorIcon.ts#L215 

I guess I was seeing missing scheme icons not channel icons.. either way... this fixes an incorrect one and makes it easier to add some missing ones for channel types currently in use

**Comment by rowanseymour** [comment]
Well these are the channels in use so I think these are the ones we want to be more specific about

**Comment by norkans7** [comment]
I was think more of removing those set to be zap as that is the default but the comment could make is clear we need to improve them

**Pull request merged**